examples/_client.rb.example

```diff
@@ -36,3 +36,5 @@ puts "Connected to OneView appliance at #{@client.url}\n\n"
 # @firmware_bundle_path = 'hp-firmware-a1b08f8a6b-HPGH-1.1.i386.rpm'
 # @sas_interconnect_type = 'Synergy 12Gb SAS Connection Module'
 # @synergy_enclosure_hostname = 'fe80::2:0:9:1%eth2'
+# @enclosure_name = '0000A66101'
+# @interconnect_name = '0000A66101, interconnect 3'
```

Comment:
I think so not.
Because that more than one interconnect can be created automatically (after add a new enclosure, for C7000), and some them can't used for create a new uplink, this iwas the reason I put the interconnect_name.
And more than one Enclosure can be added automatically (for Synergy), and it was the reason I put enclosure_name.
I'm using DCS.